app/views/analytics/_editors_list.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="chart-wrapper">
+  <div class="chart-title background-orange">
+    Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+  </div>
+  <div class="chart-stage">
+   <table class='table leader-list'>
+     <tr>
+      <th>Editor</th>
+      <th>Updates</th>
+     </tr>
+
+     <% @case_updaters_count_10.each do |who| %>
+       <% user_id = who[0].to_i %>
+       <% user_object = who[1] %>
+       <tr>
+         <% unless user_id == 0 %>
+           <td><%= @user = User.find(user_id).name %></td>
+           <td><%= link_to user_object, @user %></td>
+         <% else %>
+           <td> Anonymous </td>
+           <td><%= user_object %></td>
+         <% end %>
+       </tr>
+     <% end %>
+   </table>
+  </div>
+  <div class="chart-notes">
+    The ten most active editors over the last 30 days.
```

Comment:
Wait, this doesn't match the wording used above.  Is this the most recent editors, as in the last 10 people to edit a case?  Or is it the 10 people who made the most edits in the past 30 days?